src/components/common/ShareButton/ShareButton.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image'
+import React from 'react'
+import { AiOutlineCopy } from 'react-icons/ai'
+import { toast, ToastContainer } from 'react-toastify'
+import { ShareButtonProps } from './ShareButton.type'
+import 'react-toastify/dist/ReactToastify.css'
+
+const ShareButton: React.FC<ShareButtonProps> = () => {
```

Comment:
props特にないならこれでOK
あとButtonが複数あるコンポーネントなので複数形で

```suggestion
const ShareButtons: React.FC = () => {
```

Reply:
[73b81df](https://github.com/NUTFes/chosei-chan/pull/32/commits/73b81df898afda62ef50c5f30bc200715ae5c8dd)
[21af867](https://github.com/NUTFes/chosei-chan/pull/32/commits/21af867ea7356b2cf786ae4d1f8ae8935a7cdcfc)